app/index.js

```diff
@@ -39,30 +39,31 @@ module.exports = yeoman.generators.Base.extend({
 			type: 'confirm',
 			default: false
 		}], function (props) {
-			this.moduleName = props.moduleName;
-			this.camelModuleName = _s.camelize(props.moduleName);
-			this.githubUsername = props.githubUsername;
-			this.name = this.user.git.name();
-			this.email = this.user.git.email();
-			this.website = props.website;
-			this.humanizedWebsite = humanizeUrl(this.website);
-			this.superb = superb();
-			this.cli = props.cli;
+			var tpl = {
+				moduleName: props.moduleName,
+				camelModuleName: _s.camelize(props.moduleName),
+				githubUsername: props.githubUsername,
+				name: this.user.git.name(),
+				email: this.user.git.email(),
+				website: props.website,
+				humanizedWebsite: humanizeUrl(props.website),
+				superb: superb(),
+				cli: props.cli
+			};
 
-			this.template('editorconfig', '.editorconfig');
-			this.template('gitattributes', '.gitattributes');
-			this.template('gitignore', '.gitignore');
-			this.template('jshintrc', '.jshintrc');
-			this.template('travis.yml', '.travis.yml');
-			this.template('index.js');
-			this.template('license');
-			// needed so npm doesn't try to use it and fail
-			this.template('_package.json', 'package.json');
-			this.template('readme.md');
-			this.template('test.js');
+			this.fs.copy(this.templatePath('editorconfig'), this.destinationPath('.editorconfig'));
+			this.fs.copy(this.templatePath('gitattributes'), this.destinationPath('.gitattributes'));
+			this.fs.copy(this.templatePath('gitignore'), this.destinationPath('.gitignore'));
+			this.fs.copy(this.templatePath('jshintrc'), this.destinationPath('.jshintrc'));
+			this.fs.copy(this.templatePath('travis.yml'), this.destinationPath('.travis.yml'));
+			this.fs.copyTpl(this.templatePath('_package.json'), this.destinationPath('package.json'), tpl);
+			this.fs.copyTpl([
+				this.templatePath() + '/**',
+				'!**/{cli.js,editorconfig,gitattributes,gitignore,jshintrc,travis.yml,_package.json}'
```

Comment:
Instead of having to ignore all these, would probably be better to just copy all the files and rename them afterwards instead. It's all in memory anyways.

https://github.com/SBoudrias/mem-fs-editor#movefrom-to-options

I would also make a inline helper fn:

```js
var mv = function (from, to) {
  this.fs.move(this.templatePath(from), this.destinationPath(to));
}.bind(this);
```